README.md

```diff
@@ -123,27 +136,37 @@ Three things to be sure you establish during this phase are:
 
 Write a script to download the data (or instructions for future users on how to manually download it), and explore it. Do you understand what the columns mean? How do the three data tables relate to each other? How will you select the subset of relevant data? What kind of data cleaning is required?
 
+Make sure that you document the process for retrieving this data so it can be reproduced by a future user.
+
 It may be useful to generate visualizations of the data during this phase.
 
 ### 3. Data Preparation
 
-Through SQL and Pandas, perform any necessary data cleaning and develop a query that pulls in all relevant data for analysis in a linear regression model, including any merging of tables. Be sure to document any data that you choose to drop or otherwise exclude. This is also the phase to consider any feature scaling or one-hot encoding required to feed the data into a linear regression model.
+Through SQL and/or Pandas, perform any necessary data cleaning and develop a query that pulls in all relevant data for analysis in a linear regression model, including any merging of tables. Be sure to document any data that you choose to drop or otherwise exclude. This is also the phase to consider any feature scaling or one-hot encoding required to feed the data into a linear regression model.
 
 ### 4. Modeling
 
-Your goal here is not to build just one model; it is to build many models and then choose the best one. In this case (which is different from future machine learning models), your goal is to gain additional insight into the housing sales data from 2018 in King County, as opposed to building a generalizable model that can make predictions about unseen data.
+The modeling phase in this project is should be a brief stop-over as you are jumping back and forth between the data preparation and the evaluation phases.  If the data preparation was done correctly, it only takes a few lines of code to build a linear regression model.  Then you should be able to print out your model's metrics and quickly move to the evaluation phase.
```

Comment:
Delete the 'is': "The modeling phase in this project should be a brief stop-over ..."